TLCGen.PluggedInItems/TLCGen.Generators.CCOL/CodeGeneration/Functionality/PtpCodeGenerator.cs

```diff
@@ -184,6 +195,24 @@ public override void CollectCCOLElements(ControllerModel c)
                             _prmcmsg, k.TeKoppelenKruispunt));
                 }
             }
+
+            foreach (var k in c.PTPData.PTPKoppelingen.Where(x => (x.Dummy == true) && (x.MaakIO == true)))
```

Comment:
vraag: waarom worden de is/us niet aangemaakt als !x.Dummy ?

Reply:
Goeie vraag! De gedachte was dat de I/O alleen aangemaakt wordt wanneer er geen koppeling via PTP gewenst is, maar dat kan de gebruiker ook prima zelf beslissen.
Anderzijds: het overbrengen van in- en uitgangen van en naar hulpelementen wordt alleen gedaan bij een dummy koppeling en MaakIO. Wanneer je dan toch de IO aanmaakt bij reguliere (niet-dummy) PTP koppelingen gebeurt daar niets mee (maar je kan ze wel inkliken in de bitmap, dat levert de indruk op dat ze wel functioneel zijn).
Om die verwarrende indruk te voorkomen lijkt het me handiger om ze dan ook niet aan te maken; en gebruiker die ze toch wil kan ze zelf aanmaken via de Gebruikersopties en is er zich dan ook van bewust dat er nog wat mee moet gebeuren in de zin van koppelen aan een (regelapplicatie)functie.
Daar tegenover staat dan weer de verwarring dat je MaakIO wel kan aanvinken, maar dat dit zonder 'Dummy' geen effect heeft...